mlflow/pipelines/steps/train.py

```diff
@@ -465,6 +504,18 @@ def from_pipeline_config(cls, pipeline_config, pipeline_root):
                     )
             else:
                 step_config["using"] = "estimator_spec"
+
+            if "tuning" in step_config:
+                if "enabled" in step_config["tuning"]:
+                    step_config["tuning_enabled"] = step_config["tuning"]["enabled"]
+                else:
+                    raise MlflowException(
+                        "Tuning 'enabled' value must be set ",
```

Comment:
Can we provide a bit more context? something like:

```suggestion
                        "Tuning 'enabled' value must be specified when the 'tuning' key is included in the train step configuration.",
```